text/machine_comprehension/roberta/README.md

```diff
@@ -0,0 +1,88 @@
+|RoBERTa-BASE| [499 MB](model/roberta-base-11.onnx) |  [295 MB](model/roberta-base-11.tar.gz) |  1.6 | 11|MCC of [0.85](dependencies/roberta-sequence-classification-validation.ipynb)|
+|RoBERTa-SequenceClassification| [501 MB](model/roberta-sequence-classification-9.onnx) |  [434 MB](model/roberta-sequence-classification-9.tar.gz) |  1.4 | 9|88.5|
+
+## Source
+PyTorch RoBERTa => ONNX RoBERTa
+PyTorch RoBERTa + script changes => ONNX RoBERTa-SequenceClassification
+
+## Conversion
+Here is the [benchmark script](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/transformers/run_benchmark.sh) that was used for exporting RoBERTa-BASE model. 
+
+Tutorial for conversion of RoBERTa-SequenceClassification model can be found in the [conversion](https://github.com/SeldonIO/seldon-models/blob/master/pytorch/moviesentiment_roberta/pytorch-roberta-onnx.ipynb) notebook.
+
+Official tool from huggingface that can be used to convert transformers models to ONNX can be found [here](https://github.com/huggingface/transformers/blob/master/src/transformers/convert_graph_to_onnx.py)
+
+## Inference
+We used [ONNX Runtime](https://github.com/microsoft/onnxruntime) to perform the inference.
+
+Tutorial for running inference for RoBERTa-SequenceClassification model using onnxruntime can be found in the [inference](dependencies/roberta-inference.ipynb) notebook.
+
+### Input
+Input is a sequence of words as a string. Example: "This film is so good", tokenized by RobertaTokenizer. input_ids: Indices of input tokens in the vocabulary. It's a int64 tensor of dynamic shape (batch_size, sentence_length).
+
+### Preprocessing
+Use tokenizer.encode() to encode the input text:
+```python
+text = "This film is so good"
+tokenizer = RobertaTokenizer.from_pretrained('roberta-base')
+input_ids = torch.tensor(tokenizer.encode(text, add_special_tokens=True)).unsqueeze(0)  # Batch size 1
+```
+
+### Output
+For RoBERTa-BASE model:
+Output of this model is a float32 tensors ```[batch_size,seq_len,768]``` and ```[batch_size,768]```
+
+For RoBERTa-SequenceClassification model:
+Output of this model is a float32 tensor ```[batch_size, 2]```
+
+### Postprocessing
+For RoBERTa-BASE model:
+```
+last_hidden_states = ort_out[0]
+```
+
+For RoBERTa-SequenceClassification model:
+Print sentiment prediction
+```python
+pred = np.argmax(ort_out)
+if(pred == 0):
+    print("Prediction: negative")
+elif(pred == 1):
+    print("Prediction: positive")
+```
+
+## Dataset
+Pretrained RoBERTa-BASE model weights can be downloaded [here](https://s3.amazonaws.com/models.huggingface.co/bert/roberta-base-pytorch_model.bin)
+
+RoBERTa-SequenceClassification model weights can be downloaded [here](https://storage.googleapis.com/seldon-models/pytorch/moviesentiment_roberta/pytorch_model.bin).
+
+## Validation accuracy
+Metric and benchmarking details are provided by [fairseq](https://github.com/pytorch/fairseq/tree/master/examples/roberta)
```

Comment:
```suggestion
Metric and benchmarking details are provided by PyTorch [fairseq](https://github.com/pytorch/fairseq/tree/master/examples/roberta).
```

Reply:
Can we include some highlighted metric results in-line? It's better to include the important data directly instead of providing links.